src/games/Lexible/models/PresenterModel.ts

```diff
@@ -87,6 +87,7 @@ export const getLexiblePresenterTypeHelper = (
                 case "LetterBlockModel": return new LetterBlockModel("_");
                 case "LexiblePresenterModel": return new LexiblePresenterModel( sessionHelper, gameProps.logger, gameProps.storage);
                 case "LexiblePlayer": return new LexiblePlayer();
+                case "LexibleScoredWordMessage": return new LexibleScoredWordMessage({letters: [], score: 0, scoringPlayerId: "", sender: "", team: "", messageId: 0});
```

Comment:
This technically won't be required since we are aiming to get rid of the Ack system. We can still commit it, though, though technically the correct thing to do if we intend to continue serializing messages would be to include all of them.